samples/query_no_cache.py

```diff
@@ -26,7 +26,7 @@ def query_no_cache() -> None:
         FROM `bigquery-public-data.samples.shakespeare`
         GROUP BY corpus;
     """
-    query_job = client.query(sql, job_config=job_config)  # Make an API request.
+    query_job = client.query_and_wait(sql, job_config=job_config)  # Make an API request.
```

Comment:
Please rename this variable to `results` since it is no longer a QueryJob object.